### PR TITLE
Add page titles & descriptions

### LINKED ISSUE #186: Unique titles (and descriptions) for main nav pages
Our `<title>` is the same on every page. This will look very unpleasant in search results if we don't change it.

Meta descriptions are nice, but not as necessary. Search engines default to the first text on the page if no description is provided.

This would be a good time to integrate page-specific CSS and JS into `<head>` instead of the top of our content container too.

### PR DESCRIPTION
**Pull request by techgique**
Closes #186

### CODE REVIEW

File: app/controllers/indices_controller.rb [hunk 1]
```diff
@@ -1,21 +1,29 @@
 class IndicesController < ApplicationController
   def index
+    @title = "Index of All Names"
+    @description = ""
```
Comment (jduss4) [inline app/controllers/indices_controller.rb]: Are you waiting for Laura or somebody to add a description?

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -23,9 +23,19 @@
...
+    <% @extra_css.each do |css| %>
+  <%= css %>
+    <% end %>
+  <% end %>
 
   <!-- Scripts -->
   <%= javascript_include_tag 'application', 'data-turbolinks-track' => false %>
+  <% if !@extra_js.blank? %>
+    <% @extra_js.each do |js| %>
+  <%= js %>
```
Comment (jduss4) [inline app/views/layouts/application.html.erb]: can you make sure that this and the above <%= css %> tag are indented far enough?  small complaint, I know :)

**Comment by jduss4** [comment]
I think it looks good to me!  Good idea to put the script includes in the head, even though it reads a bit strangely from the individual templates, haha.

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (techgique) [inline app/views/layouts/application.html.erb]: This is intentional so it lines up with the indentation of the markup inside the `<head>` element. If there is a way to do it so it looks good in both places, that'd be awesome, but I like the public HTML to look nicer than indentation here if I have to choose I guess.
Though the rest of the HTML indentation is a total mess :-(

**Comment by techgique** [comment]
It does look a little strange. It could be added from the controller as well, but I figured I'd keep markup in the templates.

**Pull request merged**